src/main/resources/pl/damianszczepanik/jenkins/buildhistorymanager/model/Rule/help-matchAtMost.html

```diff
@@ -1,17 +1,13 @@
 <p><b>Description</b></p>
-<p>Number of matched builds after which the rule is not processed. Then this condition is met, only other rules are
-    processed.</p>
+<p>The rule will stop being processed after a certain number of matched builds.
+    Once this condition is met, only the other rules will continue to be processed.</p>
 
 <p><b>Use cases</b></p>
-<p>Gives possibility to limit number of changes performed by given rule
-    (what of course saves the time for long build history!).</p>
+<p>This feature provides the option to control the number of changes executed by a specific rule, which can significantly save time for longer build histories.</p>
```

Comment:
I would keep the terminology more focused: Rules process builds, not "changes". I would write it as:
```
This feature provides the option to control the number builds the rule can process.
It may save a significantly amount of time when the build history is long.
```